src/modules/transaction/components/TxComposer/index.js

```diff
@@ -40,47 +50,61 @@ const TxComposer = ({
     prioritiesLoadError,
     loadingPriorities,
   ] = useTransactionPriority();
-  const [module, command] = splitModuleAndCommand(formProps.moduleCommand);
+
   const transactionJSON = {
     module,
     command,
-    nonce: wallet.sequence?.nonce,
-    fee: 0,
-    senderPublicKey: wallet.summary?.publicKey,
+    nonce: auth?.data?.nonce,
+    fee: formProps.extraCommandFee || 0,
```

Comment:
I've notice this in multiple places that you use a number as a default value for a bigNumber. I think changing the type arbitrarily is not a good idea.